### PR TITLE
Add tests on skill management module

### PR DESCRIPTION
**Pull request by thomashoarau**
Test comportementaux RH

### CODE REVIEW

File: features/rh.feature [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    
+  Scenario: I can delete a Competence
+    Given I am logged in as "admin"
+    Given I am on "/rh/competence/modifier/14"
+    Then the response status code should be 200
+    And I press "Supprimer la compétence"
+    Then the url should match "/rh"
+    And I should not see "Django"
+    
+  Scenario: I can see parameters
```
Comment (Stoakes) [inline features/rh.feature]: Hi, 

Thank you for your PR. Can you please add `@dropSchema` annotation above this scenario ? [Example](https://github.com/n7consulting/Incipio/blob/master/features/suivi/etude_crud.feature#L13)

This what makes the [next test scenario](https://travis-ci.org/n7consulting/Incipio/builds/346728001#L3770) fail.

File: features/rh.feature [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    And I should see "Etudes liées"
+  
+  Scenario: I can edit a Competence
+    Given I am logged in as "admin"
+    Given I am on "/rh/competence/modifier/1"
+    Then the response status code should be 200
+    When I fill in "Nom" with "Test"
+    And I press "Enregistrer la compétence"
+    Then the url should match "/rh/competence/1"
+    And I should see "Test"
```
Comment (Stoakes) [inline features/rh.feature:41]: `And I should not see "PHP"`

File: features/rh.feature (hunk 1 above)
Comment (thomashoarau) [reply features/rh.feature]: Done

File: features/rh.feature (hunk 2 above)
Comment (thomashoarau) [reply features/rh.feature:41]: Done

**Comment by thomashoarau** [comment]
I didn't write the outline test but all the other routes are done. 
I can write the outline test before close the merge if you want ?

**Comment by Stoakes** [comment]
The outline test mechanics (connect as someone else than admin) are not ready yet.

Let's get this PR merged, I'll ping you when outline will be ready.
Thanks @thomashoarau

**Pull request merged**